### PR TITLE
Doc cleanup

### PR DESCRIPTION
**Pull request by yurii-prykhodko-solid**

### CODE REVIEW

File: .github/workflows/gh-pages-deploy.yaml [hunk 1]
```diff
@@ -26,7 +26,12 @@ jobs:
         run: dart pub get
 
       - name: Generate documentation files
-        run: dart run bin/generate_web_docs_content.dart -p lib/lints -o doc/docusaurus/docs -r README.md -s
+        run: >
+          dart run bin/generate_web_docs_content.dart
+            --path lib/lints
+            --docs-dir doc/docusaurus/docs/generated
+            --readme README.md
+            --sort-rules
```
Comment (yurii-prykhodko-solid) [inline .github/workflows/gh-pages-deploy.yaml:34]: @solid-daniilmarchenko it's usually better to use full arg names for CI and such -- easier to read and comprehend.

File: doc/docusaurus/docusaurus.config.js [hunk 2]
```diff
@@ -98,6 +98,7 @@ const config = {
       prism: {
         theme: lightCodeTheme,
         darkTheme: darkCodeTheme,
+        additionalLanguages: ['dart', 'yaml'],
```
Comment (yurii-prykhodko-solid) [inline doc/docusaurus/docusaurus.config.js:101]: Dart syntax highlighting.

File: .github/workflows/gh-pages-deploy.yaml (hunk 1 above)
Comment (solid-daniilmarchenko) [reply .github/workflows/gh-pages-deploy.yaml:34]: got it, thanks for advice

Comment (solid-daniilmarchenko) [reply .github/workflows/gh-pages-deploy.yaml:34]: i've decided to remove --sort-rules parameter btw cause it does not makes any sense while generating files to the folder

Comment (solid-daniilmarchenko) [inline .github/workflows/gh-pages-deploy.yaml:34]:
```suggestion
```

**Review by solid-daniilmarchenko — APPROVED** [review]

**Pull request merged**